src/Microsoft.AspNetCore.Localization/QueryStringRequestCultureProvider.cs

```diff
@@ -35,6 +35,7 @@ public override Task<ProviderCultureResult> DetermineProviderCultureResult(HttpC
             }
 
             var request = httpContext.Request;
+
```

Comment:
This isn't a matter of our coding standards because those don't mention blank lines of this type. However, blank lines before a conditional block add minimal value, much less than (say) adding them after curly braces. (Some might say they spread the code out to much and make it _harder_ to read.) They also cause a fair amount of noise in this PR. I recommend reverting all such additions.